src/inverseRobot.js

```diff
@@ -1,13 +1,17 @@
 'use strict';
 
-/*
- * @param {object} robot
- *
- * @return {object}
-*/
-
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    if (newRobot.hasOwnProperty(robot[key])) {
+      return null;
+    }
+
+    newRobot[robot[key]] = String(key);
```

Comment:
key is always string - objects can't have anything other than that - [Maps ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map)can 😄